src/client/pps/util.go

```diff
@@ -69,6 +73,60 @@ func InputCommits(input *Input) []*pfs.Commit {
 				ID:   input.Cron.Commit,
 			})
 		}
+		if input.Github != nil {
+			result = append(result, &pfs.Commit{
+				Repo: &pfs.Repo{RepoNameFromGithubInfo(input.Github.URL, input.Github.Name)},
```

Comment:
This should work like the other inputs, ie `input` should have `Repo` field on it that you use directly here and fill in, in `setPipelineDefaults`.